File: services/flashcard/src/main/kotlin/si/um/kotnik/application/dtos/FlashCardDto.kt
package si.um.kotnik.application.dtos

data class FlashCardDto(
    val id: String? = null,
    val cardSetId: String? = null,
    val title: String,
    val body: String,
    val correctAnswer: String,
    val pointsAwarded: Int = 0,
    val otherAnswers: List<String> = mutableListOf()
)

File: bff/web/WebBFF/Application/Dtos/Cards/FlashCardDto.cs
namespace WebBFF.Application.Dtos.Cards;

public class FlashCardDto
{
    public string Id { get; set; } = string.Empty;
    public string CardSetId { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public string Body { get; set; } = string.Empty;
    public string CorrectAnswer { get; set; } = string.Empty;
    public int PointAwarded { get; set; } = 0;
    public List<string> OtherAnswers { get; set; } = new();
}